test/integration/transactionsAPITests.js

```diff
@@ -283,7 +283,7 @@ describe('API Integration Tests', () => {
           .expect(201)
 
         const events = await EventModelAPI.find({})
-        events.length.should.be.exactly(6)
+        events.length.should.be.exactly(8)
```

Comment:
what changed in the events for the count to go up by 2?

Reply:
I added an orchestration to the transaction that is used in the test and that changes the number of events recorded to 8.